manifests/07-downloads-deployment.yaml

```diff
@@ -85,32 +85,36 @@ spec:
           temp_dir = tempfile.mkdtemp()
           print('serving from {}'.format(temp_dir))
           os.chdir(temp_dir)
-          for arch in ['amd64']:
+          for arch in ['amd64', 'arm64', 'ppc64le', 's390x']:
               os.mkdir(arch)
-              for operating_system in ['linux', 'mac', 'windows']:
-                  os.mkdir(os.path.join(arch, operating_system))
-          for arch in ['arm64', 'ppc64le', 's390x']:
-              os.mkdir(arch)
-              for operating_system in ['linux']:
-                  os.mkdir(os.path.join(arch, operating_system))
+              for operating_system in ['linux', 'darwin', 'windows']:
+                  os_path = os.path.join(arch, operating_system)
+                  try:
+                      os.mkdir(os_path)
+                  except OSError as error:
+                      if error.errno != errno.EEXIST:
+                          raise
+                  basename = 'oc'
```

Comment:
Not gonna lie, this is a bit uglier than the prev version. 😃

Reply:
> Not gonna lie, this is a bit uglier than the prev version.

Anything in particular that I can try to clean up?

Reply:
No, I think its fine.  Just had to refresh myself on some python-isms.

For fun, its not immediately obvious why we end with a list comprehension:
```
[Thread(i, socket=sock) for i in range(100)]
```
but not storing or using the list. 
Also followed by `time.sleep(9e9)`.  
`9e9` > `2**32` == max array length? 
But both of those have been part of this file since the beginning.  
Fun trivia 😉